_releases/release_notes_1.0.0.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Release Notes 1.0.0
+nav_order: 2
+---
+
+Babelfish Version: 1.0.0
+PostgreSQL Server Version: 13.2
+Minimum Compass Version: [1.0.0]](https://github.com/babelfish-for-postgresql/babelfish_compass/)
```

Comment:
For Compass release we can use the release link: https://github.com/babelfish-for-postgresql/babelfish_compass/releases/tag/v1.0

Reply:
Added....